Modules/ITS/src/TrendingTaskITSTracks.cxx

```diff
@@ -174,12 +174,12 @@ void TrendingTaskITSTracks::storePlots(repository::DatabaseInterface& qcdb)
         ymax = 100.;
       } else if (plot.name.find("VertexX") != std::string::npos) {
```

Comment:
why don't you put a single `else if (plot.name.find("VertexXY"))`. As said in my previous comment, you don't have anymore VertexX and VertexY but only VertexXY.

Reply:
mConfig.plots (not mConfig.datasources) is used here.
so, would merging be appropriate?

Reply:
you are right. Stupid me.